### PR TITLE
Jenkins: add ImagePullPolicy on init containers

### PR DESCRIPTION
**Pull request by ruhe**

### CODE REVIEW

**Comment by ruhe** [comment]
I signed it!

File: stable/jenkins/templates/jenkins-master-deployment.yaml [hunk 1]
```diff
@@ -28,6 +28,7 @@ spec:
               {
                   "name": "copy-default-config",
                   "image": "{{.Values.Master.Image}}:{{.Values.Master.ImageTag}}",
+                  "imagePullPolicy": {{ default "IfNotPresent" .Values.imagePullPolicy | quote }},
```
Comment (lachie83) [inline stable/jenkins/templates/jenkins-master-deployment.yaml:31]: I suggest removing the default and using the same variable as the Master container and have the default of IfNotPresent explicitly defined . Take a look at https://github.com/kubernetes/charts/pull/309

**Comment by lsjostro** [comment]
duplicate of PR #295. Should I close mine or which should we go with @lachie83?
A merge soon anyways would be awesome. :)
Cheers!

**Comment by lachie83** [comment]
Thanks @lsjostro this is indeed a dup of #295. Let's work on getting that merged.

**Pull request closed without merging**